content/applications/finance/accounting/reporting/overview/customize.rst

```diff
@@ -1,90 +1,248 @@
-**sequence number** and a **level** (Used for the line rendering).
+.. note::
+   Depending on the engine, :guilabel:`subformulas` may also be required.
 
-.. image:: customize/customize04.png
+.. image:: customize/engine-expressions.png
    :align: center
+   :alt: Engine expressions.
 
-In the **formulas** field you can add one or more formulas to assign a
-value to the balance column (and debit and credit column if applicable –
-separated by ;)
+'Odoo Domain' engine
+~~~~~~~~~~~~~~~~~~~~
 
-You have several objects available in the formula :
+With this engine, a formula is interpreted as an :ref:`Odoo domain <reference/orm/domains>`
+targeting `account.move.line` objects.
 
--  ``Ndays`` : The number of days in the selected period (for reports with a
-   date range).
+The subformula allows you to define how the move lines matching the domain are used to compute the
+value of the expression:
 
--  Another report, referenced by its code. Use ``.balance`` to get its
-   balance value (also available are ``.credit``, ``.debit`` and
-   ``.amount_residual``)
+- **sum** : The result is the sum of all the balances of the matched move lines;
 
-A line can also be based on the sum of account move lines on a selected
-domain. In which case you need to fill the domain field with an Odoo
-domain on the account move line object. Then an extra object is
-available in the formulas field, namely ``sum``, the sum of the account
-move lines in the domain. You can also use the group by field to group
-the account move lines by one of their columns.
+- **sum_if_pos** : The result is the sum of all the balances of the matched move lines if this
+  amount is positive. Else, it's `0`;
 
-Other useful fields :
+- **sum_if_neg** : The result is the sum of all the balances of the matched move lines if this
+  amount is negative. Else, it's `0`;
 
--  **Type** : Type of the result of the formula.
+- **count_rows** : The result is the number of sub-lines of this expression. If the parent line has
+  a group-by value, this will correspond to the number of distinct grouping keys in the matched move
+  lines. Else, it will be the number of matched move lines.
 
--  **Is growth good when positive** : Used when computing the comparison
-   column. Check if growth is good (displayed in green) or not.
+You can also put a `-` sign at the beginning of the subformula to **invert** the sign of the
+result.
 
--  **Special date changer** : If a specific line in a report should not use
-   the same dates as the rest of the report.
+'Tax Tags' engine
+~~~~~~~~~~~~~~~~~
 
--  **Show domain** : How the domain of a line is displayed. Can be foldable
-   (``default``, hidden at the start but can be unfolded), ``always``
-   (always displayed) or ``never`` (never shown).
+A formula made for this engine consists of a name used to match tax tags. If such tags do not exist
+when creating the expression, they will be created.
```

Comment:
@oco-odoo

Reply:
Two tags are matched by the formula. For example, if formula is A, it will require (and so, possibly create) tags +A and -A. So, plural is necessary here, but maybe remembering that here would make things clearer. 

You don't especially want to type the name of an existing tax tag here. It depends what you're doing. If your report is a more complete version of another report, you might for example want to reuse the tags of that report for the lines that are common to both. But you might just as well be creating a brand new report, and plan to configure your taxes later on with the new tags that got created.